sunlight-shadow/README.md

```diff
@@ -6,43 +6,28 @@
 * John Samuel
 * Gilles Gesquière
 
-## Sunpath tool
+## Description
 
-You can either run this script from the command line, or import it as a library into your project.
-This tool lets you calculate the sunpath for each hour of an interval of years (ex: 1975-2025), from the point of view of a single point on eath. Described by its latitde and longitude. The result is exported in a csv file, with a format compatible with [sunlight](https://github.com/VCityTeam/Sunlight) and [pySunlight](https://github.com/VCityTeam/pySunlight).
+This project aims to provide tools to calculate sunlight exposure of city objects, using data in the 3DTiles format it is composed of multiple subprojects that all work together to form a coherent toolsuite.
 
-### Requirements
+1. [Sunlight](https://github.com/VCityTeam/Sunlight) This is the C++ library that is used by pySunlight *You don´t need to worry about to use pySunlight*
+2. [pySunlight](https://github.com/VCityTeam/pySunlight)
+3. [pySunlight-Docker](https://github.com/VCityTeam/pySunlight-docker) **Use this if you don´t already know where to start**
+4. [UD-Sunlight-demo](https://github.com/VCityTeam/UD-Demo-Sunlight) A small web app for visualisation
+5. The Sunpath calculation script that is compatible with the aforementioned tools. It can be found in the SunpathTool directory.
+6. The technical repport that can be found in this folder.
 
-* Python 3.12 or higher.
-* The [pysolar](https://docs.pysolar.org/en/latest/#prerequisites-for-use) library for python.
+Each projects' github repository contains its own documentation. As well as use cases and examples.
+Each one is dependent on the previous project (pySunlight is dependent on Sunlight, and pySunlight-docker is a dockerised version of pySunlight).
 
-We recommend that you use a virtual environment befor installing dependencies
+The sunpath calculation script is optional, and only needed if you want to calculate sunlight outside of the pre-calculated 1975-2075 range.
 
-```bash
-python3 -m venv venv 
-```
+## Cloning the submodules
 
-Then activate the virtual environment and install the requirements
+This repository contains submodules, which need to be pulled (if you didn´t already specify --recusive when cloning the repo)
 
 ```bash
-. venv/bin/activate
-python3 -m pip install -r src/requirements.txt
-```
-
-**Example :**
+git clone --recursive [link] # If you haven´t cloned this repository yet
 
-Calculate sunpath for Lyon between 1975 and 2075 (inclusive)
-
-```bash
-python3 src/sunpath.py --latitude 45.75 --longitude 4.85 -s 1975 -e 2075
+git submodule update --init # If the repository is already cloned
```

Comment:
This should go above or as a separate section for running with pysunlight/docker